release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1309,6 +1313,63 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
 [id="ocp-4-17-openshift-cli-bug-fixes_{context}"]
 ==== OpenShift CLI (oc)
 
+* Previously, when using the delete functionality in the technology preview oc-mirror plug-in v2 feature to remove operator catalogs from mirror registries, the process failed with the following error:
++
+[source,terminal]
+----
+2024/08/02 12:18:03 [ERROR]: [OperatorImageCollector] pinging container registry localhost:55000: Get "https://localhost:55000/v2/": http: server gave HTTP response to HTTPS client.
+----
++
+This occurred because oc-mirror was querying the local cache using HTTPS instead of HTTP. With this update, the HTTP client is now properly configured before the query, resolving the issue.(link:https://issues.redhat.com/browse/OCPBUGS-41503[*OCPBUGS-41503*])
```

Comment:
```suggestion
This occurred because the `oc-mirror` command was querying the local cache using HTTPS instead of HTTP. With this update, the HTTP client is now properly configured before the query, resolving the issue.(link:https://issues.redhat.com/browse/OCPBUGS-41503[*OCPBUGS-41503*])
```